lib/bugsnag.rb

```diff
@@ -47,7 +47,9 @@ def configure(config_hash=nil)
 
     # Explicitly notify of an exception
     def notify(exception, overrides=nil, request_data=nil)
-      Notification.new(exception, configuration, overrides, request_data).deliver
+      notification = Notification.new(exception, configuration, overrides, request_data)
+      notification.deliver
+      notification
```

Comment:
I needed to make this change so that I could tell in the test if `notification.meta_data` had been modified.